dev-haskell/cassava/cassava-0.5.2.0.ebuild

```diff
@@ -30,15 +32,16 @@ RDEPEND=">=dev-haskell/attoparsec-0.11.3.0:=[profile?] <dev-haskell/attoparsec-0
 	>=dev-haskell/vector-0.8:=[profile?] <dev-haskell/vector-0.13:=[profile?]
 	>=dev-lang/ghc-7.8.2:=
 "
+
 DEPEND="${RDEPEND}
 	>=dev-haskell/cabal-1.18.1.3
-	test? ( <dev-haskell/hunit-1.7
-		>=dev-haskell/quickcheck-2.13 <dev-haskell/quickcheck-2.14
-		>=dev-haskell/quickcheck-instances-0.3.12 <dev-haskell/quickcheck-instances-0.4
-		>=dev-haskell/test-framework-0.8 <dev-haskell/test-framework-0.9
-		>=dev-haskell/test-framework-hunit-0.3 <dev-haskell/test-framework-hunit-0.4
-		>=dev-haskell/test-framework-quickcheck2-0.3 <dev-haskell/test-framework-quickcheck2-0.4 )
 "
+	# test? ( <dev-haskell/hunit-1.7
```

Comment:
I suggest just removing commented out depends. Comments make it harder to text-grep for upper bounds of packages.

Reply:
My thinking was that commenting the test DEPEND would make it easier to enable testing later. I have seen a few other ebuilds I'm the tree that have done this, and that did indeed make enabling tests easier for me.

Is this a hard and fast rule or simple a suggestion?

I think grep regex can pretty handily ignore these commented lines

Reply:
It's only a suggestion. If you find it useful we can keep it.

I usually use https://github.com/trofi/home/blob/master/bin/ewhat#L27 crude grep. Filtering out comments is easy there as well.

Reply:
Here's mine, I'm sort of proud of it lol. usage `grep_package binary-instances` or `grep_package dev-lang/purescript`.

I usually do `grep_package binary-instances --color=always | less -RS` actually, maybe I should make that an alias or something...

```sh
grep_package(){
    if [[ ${1} == *\/* ]]
    then
        atom="${1}"
    else
        atom="dev-haskell/${1}"
    fi

    # Skip first argument, since we used it already
    grepargs="-Prno ${@:2} --exclude-dir=metadata"
    cmd="grep ${grepargs} \"[^ ]*${atom}-\d[^ ]*(\s+<${atom}-\d[^ ]*)?\""
    echo "Executing: ${cmd}"
    eval ${cmd}
}
```